### DISCUSSION TITLE
Why `cast send` doesn't handle eip 2930 transactions ?

### DISCUSSION THREAD

**Discussion by Zylon04** [foundry-rs/foundry#8649] [Q&A] [answered]
I wanted to use cast on a node to test multiple transaction type (legacy, eip2930, eip1559) but I didn't find how to make an eip2930 transaction and nothing about access_list either. I wanted to know why does cast can't craft a transaction with access_list or an eip2930 transaction (prefixed by 1)

**Comment by zerosnacks** [comment, marked as answer]
This is now supported, see https://github.com/foundry-rs/foundry/pull/8818
